SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -671,3 +694,18 @@ void UGlobalStateManager::SetDeploymentMapURL(const FString& MapURL)
 	UE_LOG(LogGlobalStateManager, Log, TEXT("Setting DeploymentMapURL: %s"), *MapURL);
 	DeploymentMapURL = MapURL;
 }
+
+
+void UGlobalStateManager::IncrementSessionID()
```

Comment:
Hmm, initially reading the name of this function made me think it only incremented the local variable and didn't update it in Spatial. Could the updating aspect of this function be included in the name?

Reply:
✓